lib/settings/defaults.js

```diff
@@ -16,6 +16,25 @@ const filename_format = function ({testSuite = '', testCase = '', isError = fals
   return `${fileName.join('/')}${isError ? '_ERROR' : '_FAILED'}_${dateStamp}.png`;
 };
 
+const snapshot_file_format = function({testSuite = '', testCase = '', commandName, dateObject = new Date()} = {}) {
+  const fileName = [];
+  const dateParts = dateObject.toString().replace(/:/g, '').split(' ');
+  dateParts.shift();
+
+  const dateStamp = dateParts.slice(0, 5).join('-');
+  if (testSuite) {
+    fileName.push(testSuite);
+  }
+  if (testCase) {
+    fileName.push(testCase);
+  }
+  if (commandName) {
+    fileName.push(commandName);
+  }
+
+  return `${fileName.join('/')}_${dateStamp}.html`;
```

Comment:
wouldn't it be better to use https://nodejs.org/api/path.html#pathjoinpaths so we know it works on all platforms instead of using `/`

Reply:
fixed it using `path.sep`